.github/workflows/python-app.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: offer-fit-integration
```

Comment:
`offer-fit`? Is this file copied from somewhere?

Reply:
I created this for a different project and then I just updated it for invoice-utils, I changed the poetry vs pip but forget this name